scripts/build-json/build-landing-page-json.js

```diff
@@ -0,0 +1,59 @@
+const related = require('./related-content');
+const links = require('./build-link-lists');
+const markdown = require('./markdown-converter');
+
+/**
+ * Build a single list of links.
+ *
+ * These may be specified in one of three ways:
+ * - an explicitly enumerated list of pages
+ * - a reference to a "chapter_list" YAML file, which enables lists of pages to be shared
+ * - a reference to a directory, which means "include links to all the pages in this directory"
+ */
+function buildLinkList(listSpec) {
+    if (listSpec.pages) {
+        return links.linkListFromFilePaths(listSpec.title, listSpec.pages, true);
+    } else if (listSpec.chapter_list) {
+        return links.linkListFromChapterList(listSpec.chapter_list, true);
+    } else if (listSpec.directory) {
+        return links.linkListFromDirectory(listSpec.title, listSpec.directory, true);
+    }
+}
+
+/**
+ * Build a single landing page.
+ * A landing page consists of some overview text followed by one or more
+ * "link lists".
+ *
+ * A link list is just a representation of a list of links.
+ * It consists of some short text describing the list, followed by the links.
+ * Each link contains a title and a URL, and may optionally also contain
+ * a short description.
+ */
+async function buildLandingPageJSON(elementPath, data, content) {
+    const overview = {
+        type: "prose",
+        value: {
+            title: "Overview",
+            id: "overview",
+            content: await markdown.markdownToHTML(content)
+        }
+    };
+    const linkLists = await Promise.all(data.link_lists.map(async listSpec => {
+        const linkList = await buildLinkList(listSpec);
+        return {
+            type: "link_list",
+            value: linkList
+        };
+    }));
+    return {
+        title: data.title,
+        mdn_url: data.mdn_url,
+        related_content: await related.buildRelatedContent(data.related_content),
+        body: [ overview, ...linkLists]
+    };
+}
+
+module.exports = {
+    buildLandingPageJSON
+}
```

Comment:
Sorry for being a pain but I can see myself making a PR on `stumptown-content` give it the same `prettier` config that [we have in `stumptown-renderer`](https://github.com/mdn/stumptown-renderer/commit/29672d0d97951e8449c36b8f2b267ddbfebd5237). If you just make this 2-space indent that PR won't bulldozer over this file.

Reply:
Oh. I see now. Some files are 2-space. Some are 4-space. Let's just face the inevitable and make that switch some day. Ignore me for now.

Reply:
I'd happily merge a PR for this repo that would automatically fix all my formatting inconsistencies....